src/components/Footer.js

```diff
@@ -53,7 +53,7 @@ const Footer = ({background}) => (
                 </a>
             </div>
             <p className="footer__copyright"><a target="_blank" rel="noopener noreferrer" href=" http://www.eclipse.org/legal/privacy.php">Privacy Policy</a> | <a target="_blank" rel="noopener noreferrer" href=" http://www.eclipse.org/legal/termsofuse.php">Terms of Use</a> | <a target="_blank" rel="noopener noreferrer" href="http://www.eclipse.org/legal/copyright.php">Copyright Agent</a></p>
-            <p>© {(new Date()).getFullYear()} by <a href="https://typefox.io" target="_blank" rel="noopener">TypeFox</a></p>
+            <p>© {(new Date()).getFullYear()} by <a href="https://typefox.io" target="_blank" rel="noopener noreferrer">TypeFox</a></p>
```

Comment:
The canonical URL seems to be https://www.typefox.io/

Also, it's fine to have a referrer here.

Reply:
Thanks! I have removed the `noreferrer`. Sorry, I'm not able to understand the canonical is `https://theia-ide.org/`?  or do you want me to put a `/` at the end of the URL ?